spec/fixtures/prometheus/custom_nginx.template

```diff
@@ -0,0 +1,885 @@
+# This is a custom nginx configuration template for Kong specs
```

Comment:
with the plugin moved to kong repo, we can remove this custom template as it's only
needed to let prometheus test return static number of shm metrics.
it would be better to always used what kong ships for avoiding suprises